tests/integration/conftest.py

```diff
@@ -187,6 +190,68 @@ async def app(model: Model, app_no_runner: Application) -> AsyncIterator[Applica
     await model.wait_for_idle(status=ACTIVE_STATUS_NAME)
 
     # Wait until there is one runner.
-    await assert_num_of_runners(unit, 1)
+    await wait_till_num_of_runners(unit, 1)
 
     yield app_no_runner
+
+
+@pytest_asyncio.fixture(scope="module")
+async def app_scheduled_events(
+    model: Model,
+    charm_path: Path,
+    app_name: str,
+    path: str,
+    token: str,
+    http_proxy: str,
+    https_proxy: str,
+    no_proxy: str,
+) -> AsyncIterator[Application]:
+    """Application with no token.
+
+    Test should ensure it returns with the application having one runner.
+
+    This fixture has to deploy an new application. The scheduled events are set
```

Comment:
Minor typo suggestion :D
```suggestion
    This fixture has to deploy a new application. The scheduled events are set
```